webapi/src/main/scala/org/knora/webapi/slice/admin/AdminModule.scala

```diff
@@ -28,7 +27,7 @@ object AdminModule {
     AppConfig & CacheService & IriService & OntologyRepo & PredicateObjectMapper & TriplestoreService
 
   type Provided =
-    GroupService & KnoraGroupService & KnoraProjectService & KnoraUserService & MaintenanceService & PasswordService & ProjectService
+    KnoraGroupService & KnoraProjectService & KnoraUserService & MaintenanceService & PasswordService & ProjectService
```

Comment:
On this particular settings it started to work for me, otherwise I was getting the runtime error saying something like "not find implicit value for parameter...". Also the same is for `UserService` which is not presented here, but in `AdminApiModule`.

Reply:
**issue**: The `GroupService` belongs to the `domain` package , this change violates the intended module structure.

The `GroupService` belongs to the `domain` package as it encapsulating the respective domain logic for handling our groups and must therefore be provided by the `AdminModule`.

The `AdminApiModule` may depend on domain services only and provides the respective Rest* service and or routes.

Just because it "works" does not make it right. Please revert this change. Should you require support in understanding the intended architecture or fixing compile problem please reach out.

Reply:
See my comment [here](https://github.com/dasch-swiss/dsp-api/pull/3139#discussion_r1551520271)